scripts/functiona.sh

```
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+#This is function definition
+my_fun(){
+
+	echo "This is functiona definitin"
+}
+
+#This is function call
+my_fun
+
+my_function($1){
```

Shellcheck

[error] Trying to declare parameters? Don't. Use () and refer to params as $1, $2..

(SC1065)